### PR TITLE
@mzikherman => Colorized morgan logs + lib tests

### PR DESCRIPTION
**Pull request by kanaabe**
Output:
image

There's a babel-related issue when I run the whole suite locally. Maybe someone has some insight? I haven't been able to figure it out yet.

### CODE REVIEW

File: desktop/test/lib/setup.coffee [hunk 1]
```diff
@@ -0,0 +1,130 @@
+sinon = require 'sinon'
```
Comment (mzikherman) [inline desktop/test/lib/setup.coffee]: Wow, very impressive set of tests!! 🎉

File: lib/logger_format.coffee [hunk 2]
```diff
@@ -0,0 +1,16 @@
+chalk = require 'chalk'
```
Comment (mzikherman) [inline lib/logger_format.coffee]: I like this.

**Comment by mzikherman** [comment]
Looks like some real ❌ but this is amazing!

**Comment by kanaabe** [comment]
Thanks to @damassi's help, it turned out that [this test file](https://github.com/artsy/force/blob/master/desktop/test/lib/middleware/proxy_to_merged.coffee) was causing global issues when trying to run my new `setup.coffee` tests. I decided to go ahead and removed it, and related code since we don't run this split anymore. @craigspaeth, please flag if you see any problems with that otherwise this should be good to merge.

File: desktop/test/lib/setup.coffee [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        MAX_SOCKETS: 10
+        ipfilter: @ipfilter = sinon.stub().returns((req, res, next) -> next())
+        artsyPassport: @artsyPassport = sinon.stub().returns((req, res, next) -> next())
+        session: @session = sinon.stub().returns((req, res, next) -> next())
+        bodyParser:
+          json: @json = sinon.stub().returns((req, res, next) -> next())
+          urlencoded: @urlencoded = sinon.stub().returns((req, res, next) -> next())
+        bucketAssets: @bucketAssets = sinon.stub().returns((req, res, next) -> next())
+        cors: @cors = sinon.stub().returns((req, res, next) -> next())
+        artsyPassport: @artsyPassport = sinon.stub().returns((req, res, next) -> next())
```
Comment (damassi) [inline desktop/test/lib/setup.coffee]: Duplicate key

Comment (kanaabe) [inline desktop/test/lib/setup.coffee]: whoops! thank you, updated.

Comment (damassi) [inline desktop/test/lib/setup.coffee]: Not sure which editor you use, but you can install CoffeeLint directly into your editor (independent of a project-specific lintfile) to catch things like this -- thats how this jumped out :)

**Comment by kanaabe** [comment]
@mzikherman this is good to merge

**Comment by mzikherman** [comment]
🎉

**Pull request merged**